src/AspNetCore/Src/inprocessapplication.cxx

```diff
@@ -684,3 +698,83 @@ IN_PROCESS_APPLICATION::FindHighestDotNetVersion(
     // we check FAILED(hr) outside of function
     return hr;
 }
+
+//
+// Runs a standalone appliction.
+// The folder structure looks like this:
+// Application/
+//   hostfxr.dll
+//   Application.exe
+//   Application.dll
+//   etc.
+// We get the full path to hostfxr.dll and Application.dll and run hostfxr_main,
+// passing in Application.dll.
+// Assuming we don't need Application.exe as the dll is the actual application.
+//
+HRESULT
+IN_PROCESS_APPLICATION::RunStandaloneApplication()
```

Comment:
I would refactor the code to find the correct module and then from there share the execution logic. Seems like a bunch of copy pasta here.

We should also document that standalone applications when running the in process mode *does not* pick up muxer changes. This means any changes inside of `dotnet.exe` itself won't show up.